src/ui/server/controllers/insurance/business/company-details/validation/companies-house/rules/companiesHouseNumber.test.ts

```diff
@@ -0,0 +1,97 @@
+import companiesHouseNumber from './companiesHouseNumber';
+import { FIELD_IDS } from '../../../../../../../constants';
+import { RequestBody } from '../../../../../../../../types';
+import generateValidationErrors from '../../../../../../../helpers/validation';
+import { ERROR_MESSAGES } from '../../../../../../../content-strings';
+
+const {
+  EXPORTER_BUSINESS: { COMPANY_HOUSE },
+} = FIELD_IDS.INSURANCE;
+
+describe('controllers/insurance/business/company-details/validation/companies-house/rules/companiesHouseNumber', () => {
+  const mockErrors = {
+    summary: [],
+    errorList: {},
+  };
+
+  const mockBody = {
+    [FIELD_IDS.INSURANCE.EXPORTER_BUSINESS.COMPANY_HOUSE.INPUT]: '',
+  } as RequestBody;
+
+  describe('when the companies house number input is incorrectly formatted', () => {
+    it('should return validation error when number is an empty string', () => {
+      const result = companiesHouseNumber(mockBody, mockErrors);
+
+      const errorMessage = ERROR_MESSAGES.INSURANCE.EXPORTER_BUSINESS[FIELD_IDS.INSURANCE.EXPORTER_BUSINESS.COMPANY_HOUSE.INPUT].INCORRECT_FORMAT;
+      const expected = generateValidationErrors(COMPANY_HOUSE.INPUT, errorMessage, mockErrors);
+
+      expect(result).toEqual(expected);
+    });
+
+    it('should return validation error when number is less than 6 digits long', () => {
+      mockBody[FIELD_IDS.INSURANCE.EXPORTER_BUSINESS.COMPANY_HOUSE.INPUT] = '123';
```

Comment:
Could we do a 

```
const FIELD_ID = FIELD_IDS.INSURANCE.EXPORTER_BUSINESS.COMPANY_HOUSE.INPUT;
```

At the top of `describe` to avoid repitition